documentation/docs/libraries/java/how_to/_05_build_output/_code_sample.mdx

```diff
@@ -0,0 +1,15 @@
+import CodeBlock from '@theme/CodeBlock';
+import BuildAliasOutput
+    from '!!raw-loader!../../../../../../bindings/java/iota-client-java/examples/ExampleProject/src/output_builder/BuildAliasOutput.java';
+import BuildBasicOutput
+    from '!!raw-loader!../../../../../../bindings/java/iota-client-java/examples/ExampleProject/src/output_builder/BuildBasicOutput.java';
+import BuildFoundryOutput
+    from '!!raw-loader!../../../../../../bindings/java/iota-client-java/examples/ExampleProject/src/output_builder/BuildFoundryOutput.java';
+import BuildNftOutput
+    from '!!raw-loader!../../../../../../bindings/java/iota-client-java/examples/ExampleProject/src/output_builder/BuildNftOutput.java';
```

Comment:
Why do we do all the imports if we only need Basic ?

Reply:
Cause I thought the IDE would clean up the unused imports